docs/docs/introducing-jsx.md

```diff
@@ -0,0 +1,202 @@
+---
+id: introducing-jsx
+title: Introducing JSX
+permalink: docs/introducing-jsx.html
+prev: hello-world.html
+next: rendering-elements.html
+---
+
+Consider this variable declaration:
+
+```js
+let element = <h1>Hello, world!</h1>;
+```
+
+This funny tag syntax is neither a string nor HTML.
+
+It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with a full power of JavaScript.
+
+>**Note:**
+>
+>Many people get turned off by JSX the first time they see it. One might think it violates separation of concerns but we believe [otherwise](https://www.youtube.com/watch?v=x7cQ3mrcKaY).
+>
+>We encourage you to [give it five minutes](https://signalvnoise.com/posts/3124-give-it-five-minutes). Facebook and many other companies successfully use it in production both for simple and complex apps.
+>
+>Also note that JSX is optional, and you can [use React without JSX](/react/docs/react-without-jsx.html).
+
+### Embedding Expressions in JSX
+
+JSX comes with a full power of JavaScript.
+
+You can embed any [JavaScript expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Expressions) in JSX by wrapping it in the curly braces.
+
+For example, `2 + 2`, `user.name`, and `formatName(user)` are all valid expressions:
+
+```js{12}
+function formatName(user) {
+  return user.firstName + ' ' + user.lastName;
+}
+
+let user = {
+  firstName: 'Harper',
+  lastName: 'Perez'
+};
+
+let element = (
```

Comment:
I mention that

>Note that we wrapped JSX in parens and split it over multiple lines for readability.

below. I don’t think it’s worth dedicating more space to this but if you can come up with a better / more specific wording that wouldn’t be distracting, feel free to tweak here or after merging.